Sources/Commands/PackageCommands/DumpCommands.swift

```diff
@@ -70,14 +70,14 @@ struct DumpSymbolGraph: AsyncSwiftCommand {
 
         // Run the tool once for every library and executable target in the root package.
         let buildPlan = try buildSystem.buildPlan
+        let modulesGraph = try await buildSystem.getPackageGraph()
         let symbolGraphDirectory = buildPlan.destinationBuildParameters.dataPath.appending("symbolgraph")
-        let targets = try await buildSystem.getPackageGraph().rootPackages.flatMap{ $0.modules }.filter{ $0.type == .library }
-        for target in targets {
-            print("-- Emitting symbol graph for", target.name)
+        for description in buildPlan.buildModules
+            where description.module.type == .library && modulesGraph.rootPackages[description.package.id] != nil
```

Comment:
imo this is easier to read as a `guard continue`

Reply:
I had it like that initially but this `where` clause is not too complex…